lib/Commands/InterruptSignalAwaiter.h

```diff
@@ -0,0 +1,65 @@
+//===- InterruptSignalHandler.h ---------------------------------*- C++ -*-===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2015 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See http://swift.org/LICENSE.txt for license information
+// See http://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// This file implements a wrapper for handling signals and processing arrived
+// signals.
+//
+//===----------------------------------------------------------------------===//
```

Comment:
FWIW, I don't usually follow the LLVM convention of putting comments at the top of the file, I prefer putting proper documentation comments associated with actual types; those get found by automated tools.